src/redux/books/books.js

```diff
@@ -36,19 +23,31 @@ export function addNewBook(newBook) {
   };
 }
 
+export const addNewBooks = (newBook) => async (dispatch) => {
+  await ApiServices.addBook(newBook);
+  dispatch(addNewBook({ ...newBook, id: newBook.item_id }));
+};
+
 export function removeBook(bookId) {
   return {
     type: REMOVE_BOOK,
     payload: bookId,
   };
 }
 
+export const removeBooks = (id) => async (dispatch) => {
+  await ApiServices.removeBook(id);
+  dispatch(removeBook(id));
+};
+
```

Comment:
Well done 🔥 Action creator for fetching books was created and the others refactored properly